.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - '8'
+cache:
```

Comment:
Is this how one caches `yarn` dependencies? It looks from the docs like you do something like: `cache: yarn`

Reply:
You're right. It's supposed to be that way. I just used `npm` instead of `yarn` by accident!